test/fixtures/bootstrap.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Set up a super simple web server and make it accept GET and POST requests
+# for Sensu plugin testing.
+#
+
+set -e
+
+apt-get update
+
+source /etc/profile
+DATA_DIR=/tmp/kitchen/data
+RUBY_HOME=${MY_RUBY_HOME}
+
+# Set the locale
+apt-get install -y locales
+locale-gen en_US.UTF-8
+export LANG="en_US.UTF-8"
+export LANGUAGE="en_US:en"
+export LC_ALL="en_US.UTF-8"
+
+# Install beanstalkd
+apt-get install -y beanstalkd build-essential
+
+# Start beanstalkd
+service beanstalkd start
+
+cd $DATA_DIR
+SIGN_GEM=false gem build sensu-plugins-beanstalk.gemspec
```

Comment:
We do not sign our gems currently (stopped long before I became a maintainer) did this not work without? Just curious.